src/bundle/Resources/public/scss/_page-title.scss

```diff
@@ -1,41 +1,14 @@
 .ez-page-title {
-    display: grid;
-    grid-template-columns: calc(100% - #{calculateRem(50px)}) calculateRem(50px);
-    grid-column-gap: calculateRem(10px);
-    grid-template-rows: auto;
-    grid-template-areas:
-        "content-item bookmark"
-        "content-type-name .";
-
-    &__content-item {
-        width: 100%;
-        margin-bottom: 0;
-        grid-area: content-item;
-        justify-self: start;
-        display: flex;
+    &__top {
+        display: grid;
```

Comment:
Note for the future because it can be easily overlooked: this `grid` is set here so that `ellipsis` works in the title.